axiom/query/result.go

```diff
@@ -140,6 +140,9 @@ type Result struct {
 	// GroupBy is a list of field names to group the query result by. Only valid
 	// when at least one aggregation is specified.
 	GroupBy []string `json:"-"`
+	// TraceID is the ID of the trace that was generated by the server for this
+	// results query request.
+	TraceID string `json:"-"`
```

Comment:
On my side I will need this marshaled out, so I'll just use a custom result type.